app/services/getFieldName.js

```diff
@@ -0,0 +1,26 @@
+import { downloadSchemas as absDownloadSchemas } from "../app-data/abs/download-schemas";
+import { downloadSchemas as bchDownloadSchema } from "../app-data/bch/download-schemas";
+import { downloadSchemas as chmDownloadSchema } from "../app-data/chm/download-schemas";
+
+export function getFieldName(schema, fieldName, realm) {
```

Comment:
chage paramater orders order, broad => narrow
```suggestion
export function getFieldName(realm, schema, fieldName) {
```